subsys/bluetooth/audio/ascs_internal.h

```diff
@@ -7,6 +7,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#ifndef __BT_ASCS_INTERNAL_H
```

Comment:
From the C99 standard:

> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

Reply:
Changed. 

We do have **a lot** of macros in Zephyr that does not comply with this though.